ddtrace/contrib/openai/patch.py

```diff
@@ -244,6 +247,72 @@ def record_usage(self, span, usage):
             span.set_metric("openai.response.usage.%s_tokens" % token_type, num_tokens)
             self._statsd.distribution("tokens.%s" % token_type, num_tokens, tags=tags)
 
+    def generate_completion_llm_records(self, resp, span, args, kwargs):
+        # type: (Any, Span, List[Any], Dict[str, Any]) -> None
```

Comment:
The Type of response is currently Any, can we make this more defined?

Reply:
This is tough since openai can either return a json dictionary (`openai<1.0`) or a pydantic model (`openai>=1.0`). The former is a large complex dictionary which will make type hinting more gross, and the latter would be difficult to add a type hint for since this would now need to be version gated. 

I think we can ignore this for now unless you think this is a notable issue.